README.md

```diff
@@ -21,33 +21,14 @@ https://github.com/TOMP-WG/TOMP-API/tree/master/documents or through dutchmobili
 
 How can I join the TOMP working group?
 ---
-Please contact Bon Bakermans (bon.bakermans@minienw.nl). He will add you to our mailing lists and will send you the invites for the meetings.
+Please contact Bon Bakermans (aysehilal@ndw.nu). He will add you to our mailing lists and will send you the invites for the meetings.
```

Comment:
The E-Mail address of Ayse was added but the name of the contact person is still Bon Bakermans. Was this by intent?

Reply:
It looks like you also need to change Bon Bakermans into Ayse Hilal Öcal?